Client/ClientRoom.hpp

```diff
@@ -54,12 +54,13 @@ namespace client_data
         /// @brief Init some entities only created by the client.
         void _initEntities();
 
+        /// @brief Trait a disconnection request. Identify the player and add to it a disconnection component
```

Comment:
Are u sure about Trait ? How about Handle ?

Reply:
I will change that !